docs/static/releasenotes.asciidoc

```diff
@@ -30,6 +31,203 @@ This section summarizes the changes in the following releases:
+These changes result in better stability of persistent queues.
+ 
+===== Improvements to pipeline workers error handling
+
+Worker threads were not correctly monitored for a worker loop exception
+resulting in a complete logstash crash upon any exception even when multiple
+pipelines are running. Now only the failed pipeline is terminated. If pipeline
+reloading is enabled, you can edit the config and have the failed pipeline
+reloaded. 
+Implementation details are in
+https://github.com/elastic/logstash/pull/12019[#12019] and
+https://github.com/elastic/logstash/pull/12038[#12038].
+
+
+===== Improved support in App Search output
+
+We replaced the deprecated Java client library for the
+<<plugins-outputs-elastic_app_search,Elastic App Search output plugin>> with the
+Ruby client library, and expanded integration testing. These changes provide a
+foundation for expanding App Search integration and quality assurance in future
+releases. 
+
+===== Expanded API key support
+
+With this release, we've continued expanding support for {es} API keys. Support
+for API keys in the <<plugins-outputs-elasticsearch,{es} output plugin>> arrived
+in {ls} 7.8.0. {ls} 7.9.0 introduces support for {es} API keys in the
+<<plugins-inputs-elasticsearch,{es} input plugin>>, the
+<<plugins-filters-elasticsearch,{es} filter plugin>>, and {ls}
+<<ls-api-key-monitor,monitoring>> and <<ls-api-key-man,management>>. 
+
+Check out <<ls-api-keys>> for more information about using API keys with {ls}
+and {es}. 
+Implementation details are in https://github.com/elastic/logstash/pull/11953[#11953].
+
+
+===== ARM64 support (experimental)
+
+{ls} runs on arm machines! We have tested {ls} against arm64, and we are looking
+to make docker and other images available soon.
+
+ARM artifacts are not yet supported for production, and we’re offering them as
+"experimental" to early adopters.
+
+===== Performance improvement on startup and pipeline restarts
+
+This release contains several optimizations to pipeline compilation, an
+essential step of the pipeline initialization process. These changes
+significantly improve startup and pipeline-restart performance for complex
+pipelines.  
+(For technical details, check out this PR: https://github.com/elastic/logstash/pull/12060[#12060].)
+
+From our tests in three different pipelines with eight workers each, we have
+seen times decrease from 9 - 28 minutes to around 1 minute.
+
+To aid the development of pipelines, especially the performance impact of
+compilation, Logstash now reports the time taken to compile each pipeline as a
+log entry such as:
+
+[source,sh]
+-----
+[2020-08-12T14:10:29,388][INFO ][logstash.javapipeline  ][main] Pipeline Java execution initialization time {"seconds"=>0.7}
+-----
+
+
+//TODO: Figure out which featured item this belongs with
+
+* [Featured Item] 40a807d4e - do not call agent.converge_state_and_update before agent.execute (6 weeks ago) <Colin Surprenant> https://github.com/elastic/logstash/pull/12034
```

Comment:
Hmm, good question. It may belong to PQ improvements, but it is hard to follow given there's too many things linked and nothing really highlights it.